STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/az_ulib_dm_blob_ustream_interface.h

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#ifndef AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+#define AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+
+#include "az_ulib_result.h"
+#include "az_ulib_ustream.h"
+#include "nx_api.h"
+#include "nx_web_http_client.h"
+#include "stdbool.h"
```

Comment:
Please only include stdbool.h in C.
```
#ifndef __cplusplus
#include <stdbool.h>
#endif
```

Reply:
this is for the read flag `bool last_chunk` in az_blob_http_cb defined below. wouldn't we need it here?